openssl/src/pkey.rs

```diff
@@ -407,38 +408,35 @@ impl<T> Clone for PKey<T> {
 
 impl<T> PKey<T> {
     /// Creates a new `PKey` containing an RSA key.
-    #[corresponds(EVP_PKEY_assign_RSA)]
+    #[corresponds(EVP_PKEY_set1_RSA)]
     pub fn from_rsa(rsa: Rsa<T>) -> Result<PKey<T>, ErrorStack> {
```

Comment:
We should add some todo's to switch these to take `&RsaRef<T>` in the future if we're going to use these setters.

Reply:
Done